qcow2reader_test.go

```diff
@@ -0,0 +1,155 @@
+package qcow2reader
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+)
+
+type CompressionType string
+type ImageFormat string
```

Comment:
nit: use https://github.com/lima-vm/go-qcow2reader/blob/63138be42eade73a48a657be80e43e2119847a5f/qcow2reader.go#L20

Reply:
Sure, I can change it.